12-offer-encoding.md

```diff
@@ -328,6 +327,9 @@ invoices is `lnr`.
 
 1. `tlv_stream`: `invoice_request`
 2. types:
+    1. type: 1 (`payer_info`)
+    2. data:
+        * [`...*byte`:`blob`]
```

Comment:
Is the choice of an odd field deliberate aside from making this always the first field if set?

Related is there any reason there are other odd fields (e.g. `chain`) at this point?

Reply:
I was a bit nervous about using 0.  But TBH I should.  Will fix.

The only odd fields now are:

`payer_info`: (fixed to `invoice_payer_info`!) because it's actually valid to ignore this (you have to copy ignored fields anyway)
`invoice_payer_note`: same.